flightros/include/flightros/racing/racing.hpp

```diff
@@ -0,0 +1,72 @@
+#pragma once
+
+// ros
+#include <cv_bridge/cv_bridge.h>
+#include <image_transport/image_transport.h>
+#include <ros/ros.h>
+
+// standard libraries
+#include <assert.h>
+#include <Eigen/Dense>
+#include <chrono>
+#include <cmath>
+#include <cstring>
+#include <fstream>
+#include <iostream>
+#include <iterator>
+#include <sstream>
+#include <thread>
+#include <vector>
+
+// flightlib
+#include "flightlib/bridges/unity_bridge.hpp"
+#include "flightlib/bridges/unity_message_types.hpp"
+#include "flightlib/common/quad_state.hpp"
+#include "flightlib/common/types.hpp"
+#include "flightlib/objects/quadrotor.hpp"
+#include "flightlib/objects/static_gate.hpp"
+#include "flightlib/sensors/rgb_camera.hpp"
+
+// trajectory
+#include <polynomial_trajectories/minimum_snap_trajectories.h>
+#include <polynomial_trajectories/polynomial_trajectories_common.h>
+#include <polynomial_trajectories/polynomial_trajectory.h>
+#include <polynomial_trajectories/polynomial_trajectory_settings.h>
+#include <quadrotor_common/trajectory_point.h>
+
+using namespace flightlib;
+
+namespace racing {
+
+class manual_timer {
```

Comment:
ManualTimer,  do we need a class for this timer, what is the purpose?

Reply:
good point I actually didn't need it.